src/constants.ts

```diff
@@ -21,4 +26,7 @@ export const SHEET_PROPS = {
   noClickOutside: BOOL_FALSE,
   /** Removes header section, ignores #header slot */
   noHeader: BOOL_FALSE,
+  /** Loads custom animations callback */
+  handleAnimationEnter: ANIMATION_CALLBACK,
+  handleAnimationLeave: ANIMATION_CALLBACK
```

Comment:
We should allow users to write it like `@transitionEnter` and `@transitionLeave`, vue do not care about if it is prop or an emit if it is prefixed as `on`.

```suggestion
  onTransitionEnter: TRANSITION_CALLBACK,
  onTransitionEnter: TRANSITION_CALLBACK
```